CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -105,11 +135,12 @@ DEX Requirement: 6
 == Eliazar E500 Assisted-Sledgehammer $811 ==
 
 Damage: 50 + 6*STR
+Size: Two-handed
 Range: 2m
 Accuracy Modifier: +0
 STR Requirement: 6
 
-	2-handed weapon. Considered concussive (explosive) damage. Hits armor points 
+	Considered concussive (explosive) damage. Hits armor points 
 before physical damage if the target's armor has defense against explosive (i.e. 
 concussive) damage. If armor roll is greater than 50%, target rolls a DC 50% 

```

Comment:
Are there still armors that provide defense against explosive?
(this is probably scope creep. Ooops.)

Reply:
Yes, definitely scope creep. I was just killing the now-out-of-place reference to being two-handed here.